airbyte-integrations/bases/connector-acceptance-test/tools/acceptance_test_config_migration/README.md

```diff
@@ -0,0 +1,296 @@
+# Tooling for automated migration of `acceptance-test-config.yml` files
+
+This directory contains scripts that can help us manage the migration of connectors' `acceptance-test-config.yml` files.
+
+## Setup
+Before running these scripts you need to set up a local virtual environment in the **current directory**:
+```bash
+python -m venv .venv
```

Comment:
I have no preference either way since this is "Connector ops" code - happy to switch over! Especially if in the future people won't be doing much running of the scripts of their own (Ideally there's only one command to perform the testing, migrating failed configs, opening issues, etc.)

Reply:
It shouldn't be too big of a lift. But very optional, up to you in terms of how much other work you have on your plate :)